### ISSUE TITLE
Can't execute git commit when we use `pipectl event register --contexts` because of the git version

### ISSUE THREAD

**Issue by ffjlabo** [pipe-cd/pipecd#5320]
**What happened**:

In v0.49.3, we add the new feature `pipectl event register --contexts` https://github.com/pipe-cd/pipecd/issues/5028
The event watcher automatically adds the commit with the trailer.

The trailer feature is introduced from `git 2.32.o` ref: https://github.com/git/git/blob/master/Documentation/RelNotes/2.32.0.txt#L55 
~`git 2.33.0` ref: https://github.blog/open-source/git/highlights-from-git-2-33/~

But the git version in the launcher and piped container is 2.30.6.

```
docker run --rm -i -t --entrypoint /bin/sh ghcr.io/pipe-cd/launcher:v0.49.3 
/ $ git --version
git version 2.30.6
/ $ cd
~ $ mkdir test
~ $ cd test/
~/test $ git init
~/test $ touch README.md
~/test $ git add README.md
~/test $ git commit -m "Initial commit" --trailer "Test: hoge"
error: unknown option `trailer'
usage: git commit [<options>] [--] <pathspec>...
```

```
/ $ git --version
git version 2.30.6
/ $ cd
~ $ mkdir test
~/test $ git init
~/test $ touch README.md
~/test $ git add README.md
~/test $ git comit -m "test" --trailer "Test: hoge"
error: unknown option `trailer'
usage: git commit [<options>] [--] <pathspec>...
```

**What you expected to happen**:

**How to reproduce it**:

**Environment**:
- `piped` version:
- `control-plane` version:
- Others:

**Comment by ffjlabo** [comment]
Thank you for investigation 🙏  @peaceiris

**Comment by ffjlabo** [comment]
This is fixed in v0.49.4
